pkg/commands/models/commit.go

```diff
@@ -30,6 +30,14 @@ const (
 	ActionConflict = todo.Comment + 1
 )
 
+type Divergence int
+
+const (
+	DivergenceNone Divergence = iota
+	DivergenceLocal
```

Comment:
Personally I don't like the terms "ahead" and "behind" much, they are not an intuitive description of what's happening. For example, for a branch that my co-worker rebased onto a newer master, my divergence view shows my older branch commits as "ahead", but the upstream commits (which are really "newer") as "behind". I don't find this intuitive at all.

I find terms like "local/remote" (or "local/upstream") or "mine/theirs" more useful to describe the situation.

Reply:
This is still unresolved; which one should I pick?
1. DivergenceLocal/DivergenceUpstream (the current version)
2. DivergenceBehind/DivergenceAhead (your preference)
3. DivergenceMine/DivergenceTheirs
4. DivergenceLocal/DivergenceRemote

I can live with each of these, but my preference is either 3 or 4.

(We should make these names the same as what we show in the section headers.)

Reply:
Sorry, forgot to respond.

It's gonna have to be either 2 or 3 because we could be comparing any two refs. Ahead/Behind is used in vscode when it shows divergence, so there's precedence for it. The downside is that it invokes time.

Mine/Theirs is intuitive when comparing your branch to somebody else's branch, but it's confusing when you own both branches. It invokes interpersonal comparison

ChatGPT just suggested source/target. That sounds pretty neutral to me, what do you think?

Reply:
BTW as for labels: I'm happy to have the labels be something like 'Commits unique to A', and 'Commits unique to B' where we don't need to use the same terminology we're using here.

Reply:
source/target sounds _too_ neutral to me, I find it hard to relate the terms to the feature.

I agree that 'Commits unique to A' (or maybe just 'A') makes sense when we allow comparing any two refs. We don't have that feature yet, and for the specific case of comparing against upstream I prefer Local/Remote. Can we make this configurable (later, when we need it) and show Local/Remote in the headers for now?

In that case, what we use for the variable names doesn't matter too much to me. Maybe DivergenceThis/DivergenceOther. Or even DivergenceLeft/DivergenceRight?

Reply:
For sure: when comparing to upstream we can have specific headers for that.

Happy to go with DivergenceLeft/DivergenceRight as it aligns with the flag that we're passing to git log.